lib/commands/execute.js

```diff
@@ -111,11 +113,15 @@ export default {
    */
   async execute(script, args) {
     // TODO: create a type that converts args to the parameters of the associated method using the `command` prop of `executeMethodMap`
+    // this replaces "mobile:foo" with "mobile: foo"
+    if (/^mobile:\S/.test(script)) {
+      script = script.replace(/^mobile:/, 'mobile: ');
```

Comment:
I assume we should also respect scripts like `mobile:    cmd` (e.g. multiple spaces after colon) to mimic the legacy behavior

Reply:
agree with this

Reply:
not sure about trailing spaces though. the legacy logic was ignoring them as well

Reply:
okay

Reply:
oh, I'll have it trim